### ISSUE TITLE
Autumn 2017

### ISSUE THREAD

**Issue by ashper** [erengy/taiga#472]
Any word on the Autumn 2017 anime being added?

Cheers

**Comment by tophf** [comment]
My script in #390 still works on MAL.

Additionally, if you watch gintama#6, add this to anime-relations.txt somewhere in db directory:

```
# Gintama -> ~. Porori-hen
- 918|818:329-? -> 35843|13726:1-?!
```

**Comment by ashper** [comment]
Works for me, I now have the fall season loaded, cheers @tophf

**Comment by deusxanime** [comment]
Thx for the script @tophf, but would be nice for it to not be necessary. I see I was also on that thread for that same reason as well previously...

What do you run your script/code in that is mentioned in that other issue? Is it Windows or Linux code? Just at cmd/shell prompt? 

Oddly you say to put the generated xml in the data/db/season folder, but I don't have any 2017 xml files in there currently for the other seasons this year. Should I? Worked fine downloading things for Summer season.

@erengy Will 1.3.0 automate this to save you the effort and us the inpatience, and if so, release anytime soon?

**Comment by tophf** [comment]
That's a script you run in browser console (Ctrl-Shift-I or F12 key). The script copies the results to clipboard. As for the files, do a global disk file search.

**Comment by erengy** [comment]
Fall 2017 season is now available for v1.2.5 and v1.3.0-beta users. If it doesn't appear automatically at the Seasons page, try using the *Help → Check for updates* menu item.

@deusxanime Manual compilation of season data will be necessary for v1.3 as well, until MyAnimeList updates their API.

**Comment by deusxanime** [comment]
@erengy bummer... thanks for your manual efforts then!
